tests/cephadm/test_package_validation.py

```diff
@@ -0,0 +1,162 @@
+import re
+
+from cli.utilities.containers import Container
+from cli.utilities.packages import Package, PackageError, ReposError, Rpm
+from cli.utilities.utils import get_custom_repo_url
+from utility.log import Log
+
+log = Log(__name__)
+
+
+YUM_REPO_DIR = "/etc/yum.repos.d"
+
+installed_packages = [
+    "chrony",
+    "lvm2",
+    "podman",
+]
+
+packages_to_install = [
+    "cephadm-ansible",
+]
+
+rpm_dependent_packages = [
+    "ansible-core",
+    "ansible-collection-ansible-posix",
+    "ansible-collection-community-general",
+]
+
+rpm_client_packages = [
+    "ceph-common",
+    "ceph-fuse",
+    "cephfs-top",
+]
+
+
+def validate_rpm_version(node, package_list, version_config):
```

Comment:
Create this method for single package instead of list

Reply:
Add method as compare here [1], which will return -1, 0, 1 based on comparision.

[1] https://github.com/red-hat-storage/cephci/blob/master/cli/utilities/packages.py#L23

Reply:
I didn't understand why we needed to do it for a single package.

As per the test requirement, this method makes it easy to use with package_list as we are assigning most of the packages in the list.

Reply:
this is comparision and for each package same steps need to be repeated. Instead loop inside method, loop over the method.

As this is specific to comparing rpm package version, we already have class Package which perform respective operations.

Add same steps to compare version and return below codes as described -

1. `-1` if version is less than expected
2. `0` if version is exactly same
3. `1` if version is more than expected